### PR TITLE
Add new "schemaName" option to support postgresql schemas

### PR DESCRIPTION
**Pull request by sgress454**

### CODE REVIEW

File: sequel/index.js [hunk 1]
```diff
@@ -186,17 +188,18 @@ Sequel.prototype.create = function create(currentTable, data) {
  */
 
 Sequel.prototype.update = function update(currentTable, queryObject, data) {
-
+  console.log("currentTable", currentTable);
```
Comment (devinivy) [inline sequel/index.js]: Just flagging for removal!

File: sequel/select.js [hunk 2]
```diff
@@ -14,6 +14,9 @@ var hop = utils.object.hasOwnProperty;
 
 var SelectBuilder = module.exports = function(schema, currentTable, queryObject, options) {
 
+  // if (options.schemaName && currentTable.indexOf('.') > -1) {
```
Comment (devinivy) [inline sequel/select.js]: Can this be removed?

**Comment by sgress454** [comment]
Removed offending codez, thanks @devinivy

**Comment by sgress454** [comment]
I have a companion branch ready to go on https://github.com/balderdashy/sails-postgresql/tree/schema, but waiting to run tests.

**Comment by particlebanana** [comment]
Thanks @sgress454

**Pull request merged**